docs/resources/recipient.md

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_recipient Resource
+
+Within a metastore, Unity Catalog provides the ability to create a recipient to attach delta shares to.
+
+A `databricks_recipient` is contained within [databricks_metastore](metastore.md) and can contain a list of shares.
+
+## Example Usage
+
+```hcl
+resource "databricks_recipient" "db2open" {
```

Comment:
@wchau - what do you think about `databricks_catalog_recipient` or `databricks_share_recipient` as alternative logical names for this?

Reply:
`databricks_catalog_recipient` would not make sense in the Delta Sharing context. What's wrong with `databricks_recipient`? In UC and DBR, `recipient` is a top-level keyword.